webapp/tests/karma/ts/services/format-date.service.spec.ts

```diff
@@ -2,10 +2,12 @@ import { TestBed } from '@angular/core/testing';
 import sinon from 'sinon';
 import { expect } from 'chai';
 import * as moment from 'moment';
+import * as BikramSambat from 'bikram-sambat';
 
 import { FormatDateService } from '@mm-services/format-date.service';
 import { SettingsService } from '@mm-services/settings.service';
 import { TranslateService } from '@ngx-translate/core';
+import { LanguageService } from '@mm-services/language.service';
```

Comment:
This was already there but if you have time, can you pls move `import { TranslateService } from '@ngx-translate/core';` to the first block of imports?